htdocs/display.php

```diff
@@ -28,8 +28,14 @@
     $result = $ldap_connection[1];
 
     # Find object type
-    if ( preg_match( '/'.$ldap_group_base.'$/i', $dn) ) { $type = "group"; }
-    else { $type = "user"; }
+    if (isset($_POST['type'])) { $type = $_POST['type']; }
+    } else if (isset($ldap_user_regex)) {
+	if ( preg_match( '/'.$ldap_user_regex.'$/i', $dn) ) { $type = "user"; }
```

Comment:
This is wrong, DN value does not end with the regexp value. I prefer to define the full regex in configuration.
```suggestion
	if ( preg_match($ldap_user_regex, $dn) ) { $type = "user"; }
```